contrib/IECoreMantra/src/IECoreMantra/ProceduralPrimitive.cpp

```diff
@@ -179,7 +184,8 @@ void IECoreMantra::ProceduralPrimitive::addVisibleRenderable( VisibleRenderableP
 			addGeometry(gdp, 0.0f);
 			while ( !m_renderer->m_motionTimes.empty() )
 			{
-				setPreTransform( convert< UT_Matrix4T<float> >(m_renderer->m_motionTransforms.front()),
```

Comment:
Does H15 still require the float matrix?

Can we just convert directly to double `convert< UT_Matrix4T<double> >` rather than converting to float and constructing another UT_Matrix afterwards?